webapp/channels/src/components/invitation_modal/result_table.test.tsx

```diff
@@ -91,7 +91,7 @@ describe('ResultTable', () => {
     test('user invites render as users', () => {
         props.rows = [{
             user: defaultUser,
-            reason: 'added successfuly',
+            reason: {id: 'success', defaultMessage: 'added successfully'},
```

Comment:
I see you corrected the string. Can you update also `en.json`? Or was it already correct in `en.json` and that is why this was updated?

Reply:
This is just a test file, so these don't need to go into `en.json`